src/TestInfrastructure/VsObjectModelMocks/ProjectMock.cs

```diff
@@ -37,6 +37,8 @@ public class ProjectMock : VsUIHierarchyMock, IVsProject, Project, IVsBuildPrope
 
         public DTE DTE { get; set; }
 
+        public object Project { get; set; }
```

Comment:
@duncanp-sonar I'm using this to set it to VCProjectMock in the tests.
VCProjectMock is not a complete mock, so it didn't feel right to move it to Infrastructure.